### PR TITLE
Set-Cookie:  Domain attribute clarification

### PR DESCRIPTION
**Pull request by gusega**
describe in positive way (what can)

### Description

Describe in positive way (what can) to make doc more clear.

### CODE REVIEW

**Comment by Josh-Cena** [comment]
I'm not sure if you can set cookies to its parent domain, so I'll let @bsmth take a look

File: files/en-us/web/http/cookies/index.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ The `Domain` and `Path` attributes define the _scope_ of a cookie: what URLs the
   Set-Cookie: id=a3fWa; Expires=Thu, 21 Oct 2021 07:28:00 GMT; Secure; HttpOnly; Domain=mozilla.org
   ```
 
-  If the `Set-Cookie` header does not specify a `Domain` attribute, the cookies are available on the server that sets it _but not on its subdomains_. Therefore, specifying `Domain` is less restrictive than omitting it. However, it can be helpful when subdomains need to share information about a user. Note that you can't set a different domain to the one the header is set from, or one of its subdomains; see [Invalid domains](/en-US/docs/Web/HTTP/Headers/Set-Cookie#invalid_domains) for more details.
+  If the `Set-Cookie` header does not specify a `Domain` attribute, the cookies are available on the server that sets it _but not on its subdomains_. Therefore, specifying `Domain` is less restrictive than omitting it. However, it can be helpful when subdomains need to share information about a user. Note that you can only set a domain to the one the header is set from or to its parent domain; see [Invalid domains](/en-US/docs/Web/HTTP/Headers/Set-Cookie#invalid_domains) for more details.
```
Comment (hamishwillee) [inline files/en-us/web/http/cookies/index.md:157]: This is correct but perhaps we can be more explicit (and dumping the bit about why setting domain is useful at this point).

```suggestion
  If the `Set-Cookie` header does not specify a `Domain` attribute, the cookies are available on the server that sets it _but not on its subdomains_. Therefore, specifying `Domain` is less restrictive than omitting it.
  Note that a server can only set the `Domain` attribute to its own domain or a parent domain, not to a subdomain or some other domain.
  So, for example, a server with domain `foo.example.com` could set the attribute to `example.com` or `foo.example.com`, but not `bar.foo.example.com` or `elsewhere.com` (the cookies would still be _sent_ to subdomains such as `bar.foo.example.com` though).
See [Invalid domains](/en-US/docs/Web/HTTP/Headers/Set-Cookie#invalid_domains) for more details.
```

This falls out of this section of the spec:

>    The user agent will reject cookies unless the Domain attribute specifies a scope for the cookie that would include the origin server.  
> For example, the user agent will accept a cookie with a Domain attribute of "example.com" or of  foo.example.com" from foo.example.com, but the user agent will not accept a cookie with a  Domain attribute of bar.example.com" **or of "baz.foo.example.com"**.

Comment (bsmth) [reply files/en-us/web/http/cookies/index.md:157]: Good call, linking here for convenience: https://datatracker.ietf.org/doc/html/rfc6265#section-4.1.2.3

**Review by bsmth — APPROVED** [review]
Thank you all! I think this section would be better as paragraphs and not list items, but we can have a look at formatting later, so we should merge this one now. 🚢

**Pull request merged**